src/services/sdk.ts

```diff
@@ -181,7 +184,7 @@ const dispatchProfile = (
 
   dispatch(
     upsertProfile({
-      ...profile.describes,
```

Comment:
Not sure what this was supposed to be.

Reply:
Previously, `"Profile"` activity content objects had a nested `"Person"` object under the describe key with the username on it, but they're the same thing now.